### PR TITLE
Task 1 a

### PR DESCRIPTION
**Pull request by AndriiSkoromnyi**
Теперь должно работать и должны быть видны изменения

### CODE REVIEW

File: SampleHierarchies.Gui/MainScreen.cs [hunk 1]
```diff
@@ -1,4 +1,10 @@
+<<<<<<< Updated upstream
```
Comment (pyskunov) [inline SampleHierarchies.Gui/MainScreen.cs:1]: Git conflict

File: SampleHierarchies.Interfaces/Services/ISettingsService.cs [hunk 2]
```diff
@@ -1,13 +1,15 @@
-﻿using SampleHierarchies.Interfaces.Data;
+﻿using System.Drawing;
```
Comment (pyskunov) [inline SampleHierarchies.Interfaces/Services/ISettingsService.cs:1]: we won't use `System.Drawing;`, remove

File: SampleHierarchies.Interfaces/Services/ISettingsService.cs [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+using SampleHierarchies.Interfaces.Data;
 
 namespace SampleHierarchies.Interfaces.Services
 {
     public interface ISettingsService
     {
-        // Метод для чтения данных о настройках из файла JSON
+        Color MainScreenTextColor { get; set; }
+        ISettings Settings { get; set; } // Existing property
```
Comment (pyskunov) [inline SampleHierarchies.Interfaces/Services/ISettingsService.cs:9]: Why do we have both ` ISettings Settings` and `ISettings _settings`?

Only one is required

File: SampleHierarchies.Interfaces/Services/ISettingsService.cs [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+        ISettings _settings { get; set; } // New property
+        void ChangeTextColors(ConsoleColor textColor);
```
Comment (pyskunov) [inline SampleHierarchies.Interfaces/Services/ISettingsService.cs:11]: This has to be removed. Not a part of Task 1A

File: SampleHierarchies.Services/SettingsService.cs [hunk 5]
```diff
@@ -1,35 +1,51 @@
 ﻿using Newtonsoft.Json;
+using SampleHierarchies.Data;
 using SampleHierarchies.Interfaces.Data;
 using SampleHierarchies.Interfaces.Services;
+using System;
+using System.Drawing;
```
Comment (pyskunov) [inline SampleHierarchies.Services/SettingsService.cs:6]: We won't use `System.Drawing`, remove

File: SampleHierarchies.Services/SettingsService.cs [hunk 6]
```diff
@@ -1,35 +1,51 @@
...
 
 namespace SampleHierarchies.Services
 {
     public class SettingsService : ISettingsService
     {
+        public Color MainScreenTextColor { get; set; }
+        public ISettings Settings { get; set; }
+        public ISettings _settings { get; set; } // Add this line
+
+        public void ChangeTextColors(ConsoleColor textColor)
```
Comment (pyskunov) [inline SampleHierarchies.Services/SettingsService.cs:17]: Not a part of Task 1A, remove

File: SampleHierarchies.Interfaces/Services/ISettingsService.cs (hunk 2 above)
Comment (AndriiSkoromnyi) [reply SampleHierarchies.Interfaces/Services/ISettingsService.cs:1]: System.Drawing used in Color MainScreenTextColor, we can't remove it

**Pull request merged**